src/overlays/actors/ovl_En_Tanron5/z_en_tanron5.c

```diff
@@ -28,32 +35,493 @@ const ActorInit En_Tanron5_InitVars = {
     (ActorFunc)EnTanron5_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BE5DA4 = {
-    { COLTYPE_NONE, AT_ON | AT_TYPE_ALL, AC_ON | AC_TYPE_PLAYER | AC_TYPE_ENEMY | AC_TYPE_OTHER, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK2, { 0xF7CFFFFF, 0x00, 0x00 }, { 0xF7CFFFFF, 0x00, 0x00 }, TOUCH_ON | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_ON | AT_TYPE_ALL,
+        AC_ON | AC_TYPE_PLAYER | AC_TYPE_ENEMY | AC_TYPE_OTHER,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK2,
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_ON | TOUCH_SFX_NORMAL,
+        BUMP_ON,
+        OCELEM_ON,
+    },
     { 70, 450, 0, { 0, 0, 0 } },
 };
 
-#endif
+f32 D_80BE5DD0 = 1.0f;
+
+Vec2s D_80BE5DD4[] = {
+    { 0x4B0, 0x9C4 },  { -0x4B0, 0x9C4 },  { 0x4B0, -0x9C4 },  { -0x4B0, -0x9C4 }, { 0x9C4, 0x4B0 },
+    { -0x9C4, 0x4B0 }, { 0x9C4, -0x4B0 },  { -0x9C4, -0x4B0 }, { 0x3E8, 0x3E8 },   { -0x3E8, 0x3E8 },
+    { 0x3E8, -0x3E8 }, { -0x3E8, -0x3E8 }, { 0x000, -0x3E8 },  { 0x000, 0x3E8 },   { 0x3E8, 0x000 },
+    { -0x3E8, 0x000 }, { 0x000, -0x7D0 },  { 0x000, 0x7D0 },   { 0x7D0, 0x000 },   { -0x7D0, 0x000 },
+};
+
+Gfx* D_80BE5E24[] = {
+    object_boss02_DL_007D18, object_boss02_DL_007D18, object_boss02_DL_007D18, object_boss02_DL_007D18,
+    object_boss02_DL_007D18, object_boss02_DL_007D18, object_boss02_DL_007D18, object_boss02_DL_007D18,
+    object_boss02_DL_007D18, object_boss02_DL_007D18, object_boss02_DL_007D18, object_boss02_DL_007D18,
+    object_boss02_DL_006FD0, object_boss02_DL_006FD0, object_boss02_DL_006FD0, object_boss02_DL_006FD0,
+    object_boss02_DL_006FD0, object_boss02_DL_006FD0, object_boss02_DL_006FD0, object_boss02_DL_006FD0,
+};
+
+f32 D_80BE5E74[] = {
+    0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f,
+    0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f, 0.09f,
+};
+
+typedef struct {
+    /* 0x00 */ Vec3f unk_00;
+    /* 0x0C */ f32 unk_0C;
+    /* 0x10 */ f32 unk_10;
+    /* 0x14 */ f32 unk_14;
+    /* 0x18 */ Vec3f unk_18;
+    /* 0x24 */ u8 unk_24;
+    /* 0x26 */ s16 unk_26;
+    /* 0x28 */ char unk28[0x4];
+    /* 0x2C */ s16 unk_2C;
+    /* 0x2E */ char unk2E[0x6];
+    /* 0x34 */ f32 unk_34;
```

Comment:
```suggestion
    /* 0x26 */ s16 unk_26;
    /* 0x28 */ UNK_TYPE1 unk28[0x4];
    /* 0x2C */ s16 unk_2C;
    /* 0x2E */ UNK_TYPE1 unk2E[0x6];
    /* 0x34 */ f32 unk_34;
```

Could these pads be removed at all?